as.c

```diff
@@ -3,6 +3,7 @@
 #include <string.h>
 
 #include "vm.h"
+#include "opcode.h"
```

Comment:
There is a miss in Makefile, it will now generate and recompile all file with `$ make`.

I can not figure out how to let makefile do as:
    1. check `opcode.h`, if outdate or not exist then `regen-opcode`
    2. compile as usual